data/themes/classic/style.css

```diff
@@ -585,10 +585,6 @@ SideBar QToolButton {
 
 /* Instrument plugin list */
 
-PluginDescList {
-	background-color: #5b6571;
-}
-
```

Comment:
Just making sure, this is unused now?

Reply:
The class has been removed in 3ef33dbbe994141ce3d7c5da7b9d44bcb1fdffb2, and the color scheme after that commit has been accepted (See description of 8e2c586ea87806b10edad385e257fd85f52341a0). So it is unused. Also, executing LMMS with this PR, it looks unchanged.